test/scripts/base-host-check.sh

```diff
@@ -169,4 +169,12 @@ if (( $# > 0 )); then
     if jq -e '.blueprint.enabled_modules' "${config}" || jq -e '.blueprint.packages[] | select(.name | startswith("@") and contains(":")) | .name' "${config}"; then
         check_modularity "${config}"
     fi
+
+    if jq -e '.blueprint.customizations.hostname' "${config}"; then
+        [[ "$(hostname)" == "$(jq -r '.blueprint.customizations.hostname' "${config}")" ]]
+    fi
+
+    if jq -e '.blueprint.customizations.kernel.name' "${config}"; then
```

Comment:
(super nitpick) maybe a small comment that the kernel-rt,debug customization sets this (its not part of all-configs) but maybe/probably overkill

Reply:
Not sure what you mean.  Yeah, we don't set any alt kernels in all-customizations.  Maybe we should make it `kernel-debug` though?

We have a `kernel-debug` config for `iot-commit`.  Maybe we should add `ami` to that config too to cover it.

Reply:
I added `kernel-debug` as the kernel name in `all-customizations`, making the name of the test a little more accurate.